tasks-by-users/Tauqeer-Ahmed-99/First Webpage/index.html

```diff
@@ -0,0 +1,88 @@
+<!doctype html>
+<html>
+    <head>
+        
+        <!-- 
+        Created & Owned by Tauqeer Khan
+        Date:16/07/21
+        -->
```

Comment:
also maintain it with prettier

Reply:
done with prettier changes... @faran4engg